src/pages/LiveChat/components/messages/MessagesBar.tsx

```diff
@@ -11,7 +11,8 @@ type Props = {
 export default function MessagesBar({ categoryId, messagesListDb, messagesListActionRef }: Props) {
   const { user } = useOutletContext() as LiveChatContext;
 
-  const [messageContent, setMessageContent] = useState("");
+  //SG If you use typescript is recommanded to also add styles to useState
+  const [messageContent, setMessageContent] = useState<string>("");
```

Comment:
Here, it's the same thing with and without <string> as the default state is "" and the type is automatically deduced.